Various technical notes/transition_infos.json

```
@@ -305,11 +305,6 @@
           "area_name": "The Great Tree",
           "requires": []
         },
```

Altar does lead to the St clair boss fight though
```suggestion
        },
        {
          "area_id": "0xB8D5CE86",
          "area_name": "Mouth of Inti (2 exits)",
          "requires": []
        },
```

---

As mentioned in both the commit titles as well as the utils.py file that writes it to the Spoiler Log:
**Mouth of Inti** is temporarily removed from the randomization proccess. An essential part of that removal is that this exit is not included in the .json file.

---

I mark this as resolved because it's a duplicate of another conversation.